### PR TITLE
[FIX] account: TestAccountComposerPerformance ubl

### PR DESCRIPTION
**Pull request by Levizar**
The aim of this commit is to make `TestAccountComposerPerformance` more robust.

Context:
In `TestAccountComposerPerformance` (account module), we are writing
on the field `checkbox_ubl_cii_xml`.
This field is instantiate by the `account_edi_ubl_cii` module which
might not be present.
This module get installed when `account` and `base_vat` are installed but
`base_vat` is conditionally installed.
(see `account/__init__.py` for the details)

Before the commit:
`TestAccountComposerPerformance` could fail in "weird" circumstance.

After the commit:
`TestAccountComposerPerformance` is safer

task-id: None

### CODE REVIEW

**Comment by william-andre** [comment]
@robodoo r+

File: addons/account/tests/test_account_move_send.py [hunk 1]
```diff
@@ -331,11 +323,9 @@ def test_move_composer_single_lang(self):
 
         default_ctx = test_move.action_send_and_print()['context']
         default_ctx['default_mail_template_id'] = move_template.id
```
Comment (smetl) [inline addons/account/tests/test_account_move_send.py:325]: remove ?

**Comment by william-andre** [comment]
Because it is causing issues everyday on the nightly and we are asked to clean it.
We can clean further if you want to but there is no need to delay the real fix for some minor refactoring

**Comment by smetl** [comment]
@william-andre And so what? Be a bit more social and speak to your colleagues instead of acting like an asshole -_-

**Comment by smetl** [comment]
@robodoo r+

**Comment by william-andre** [comment]
:heart:

**Pull request closed without merging**